src/lib/common.js

```diff
@@ -101,10 +101,27 @@ const getConfigEnv = async options =>
     options
   )
 
+const sanitizeConfig = config => {
+  // recurse configuration objects and arrays to remove whitespace
+  const sanitize = (chunk) => {
+    return _.reduce(chunk, (result, v, k) => {
+      if (_.isObject(v) || _.isArray(v)) {
+        result[k] = sanitize(v);
```

Comment:
This is where we recursively call this function to sanitize nested objects and arrays.